x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/notes_tab.cy.ts

```diff
@@ -30,15 +30,14 @@ import {
 } from '../../../tasks/timeline';
 
 import { TIMELINES_URL } from '../../../urls/navigation';
+import { deleteTimelines } from '../../../tasks/api_calls/common';
 
 const text = 'system_indices_superuser';
 const link = 'https://www.elastic.co/';
 
 describe.skip('Timeline notes tab', { tags: ['@ess', '@serverless'] }, () => {
-  before(() => {
-    login();
-    visit(TIMELINES_URL);
-
+  beforeEach(function () {
+    deleteTimelines();
```

Comment:
same note here, is `deleteTimelines` really necessary here? Especially with the test being skipped, this change will not even be applied... If someone takes on the refactor of this file to unskip it one day, they might change it drastically so I feel like this change is made too early